### ISSUE TITLE
Memory Leak on Build

### ISSUE THREAD

**Issue by just-maik** [nuxt-themes/docus#827]
Hello, today I've setup up a docus project using `npx nuxi@latest init docs -t themes/docus`
after `yarn install` 'ing I wanted to build using `yarn build` but it just got stuck prerendering routes.
Because I changed some stuff I thought this is an Issue on my end, but I tested it in on a fresh install and got the following Stacktrace

```
ℹ Prerendering 4 routes                                                        nitro 08:07:41
  ├─ /api/_content/cache.1676966816530.json (67ms)                             nitro 08:07:41
 
<--- Last few GCs --->

[18418:0x5ae2d60]   450908 ms: Mark-sweep 3965.4 (4133.2) -> 3953.0 (4134.2) MB, 3608.0 / 0.3 ms  (average mu = 0.169, current mu = 0.096) task scavenge might not succeed
[18418:0x5ae2d60]   455008 ms: Mark-sweep 3966.2 (4134.2) -> 3954.3 (4135.2) MB, 3758.6 / 0.3 ms  (average mu = 0.128, current mu = 0.083) task scavenge might not succeed


<--- JS stacktrace --->

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
 1: 0xb06730 node::Abort() [/usr/bin/node]
 2: 0xa1b6d0  [/usr/bin/node]
 3: 0xce1e60 v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [/usr/bin/node]
 4: 0xce2207 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [/usr/bin/node]
 5: 0xe99875  [/usr/bin/node]
 6: 0xe9a356  [/usr/bin/node]
 7: 0xea887e  [/usr/bin/node]
 8: 0xea92c0 v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [/usr/bin/node]
 9: 0xf0c704 v8::internal::ScavengeJob::Task::RunInternal() [/usr/bin/node]
10: 0xd9fe1b non-virtual thunk to v8::internal::CancelableTask::Run() [/usr/bin/node]
11: 0xb726a4  [/usr/bin/node]
12: 0xb75c87 node::PerIsolatePlatformData::FlushForegroundTasksInternal() [/usr/bin/node]
13: 0x154ca06  [/usr/bin/node]
14: 0x155f134  [/usr/bin/node]
15: 0x154d358 uv_run [/usr/bin/node]
16: 0xa43975 node::SpinEventLoop(node::Environment*) [/usr/bin/node]
17: 0xb487e6 node::NodeMainInstance::Run() [/usr/bin/node]
18: 0xacd2ac node::Start(int, char**) [/usr/bin/node]
19: 0x7f2eea062d90  [/lib/x86_64-linux-gnu/libc.so.6]
20: 0x7f2eea062e40 __libc_start_main [/lib/x86_64-linux-gnu/libc.so.6]
21: 0xa4080c  [/usr/bin/node]
Aborted (core dumped)
```
To replicate just open up the starter on StackBlitz using the button on the getting-started page or [this](https://stackblitz.com/github/nuxt-themes/docus-starter) and run yarn build after init.
Keep in mind I waited several minutes for this to stacktrace on a pod.

**Comment by Tahul** [comment]
@farnabaz ; could this be related to content?

**Comment by just-maik** [comment]
Fixed with nitro@2.2.3

**Comment by Tahul** [comment]
💪 ; thanks for reporting @just-maik
